### DISCUSSION TITLE
GitHub Education Verification Process keeps rejecting with valid documents.

### DISCUSSION THREAD

**Discussion by saifbashar** [community/community#60674] [GitHub Education]
### Hi everyone,

Hi,
I already submitted all of my document including my university id card, university registration card, university class schedule with my edu email but every time i was rejected. What's wrong , yes all of my document doesn't have a valid end data[Our university id card or registration card doesn't have issue end data or valid till date],

What can i do then?

Please review all of my documents i already submitted then verify me if i am student or not. I was contract them through support ticket my their reply like a bot.
![Uploading image.png…]()

Thanks

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. We can confirm that you have since been successful - enjoy your journey through GitHub Education.
